### PR TITLE
Implementing the Reader.

### PR DESCRIPTION
**Pull request by barakbasson**

### CODE REVIEW

File: src/utils/reader.py [hunk 1]
```diff
@@ -1,3 +1,223 @@
...
+            endianness="<",
+        )
+        self.gender: str = from_bytes(
+            data=self.file.read(CHAR_SIZE_IN_BYTES),
+            data_type="char",
+            endianness="<",
+        )
+
+
+class Snapshot:
```
Comment (OfekBasson) [inline src/utils/reader.py:84]: Please add documentation for the snapshot class.

**Review by OfekBasson — COMMENTED** [review]
Finished reviewing

File: src/utils/reader.py (hunk 1 above)
Comment (barakbasson) [reply src/utils/reader.py:84]: It will be done in a different PR.

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ script:
   - pytest --cov
   - flake8 --suppress-none-returning --ignore ANN101 src
   - flake8 --suppress-none-returning --ignore ANN101 ./*.py
+  - flake8 --suppress-none-returning --ignore ANN101 tests
```
Comment (OfekBasson) [inline .travis.yml]: Non-relevant

**Pull request merged**